homeassistant/components/generic/config_flow.py

```diff
@@ -194,6 +194,14 @@ async def async_test_stream(hass, info) -> dict[str, str]:
     """Verify that the stream is valid before we create an entity."""
     if not (stream_source := info.get(CONF_STREAM_SOURCE)):
         return {}
+    if not isinstance(stream_source, template_helper.Template):
+        stream_source = cv.template(stream_source)
+        stream_source.hass = hass
```

Comment:
```suggestion
        stream_source = template_helper.Template(stream_source, hass)
```

Can we use the helper directly here?

Reply:
Agreed, done. 3a994b8